execution_engine/src/core/engine_state/genesis.rs

```diff
@@ -1145,8 +1145,8 @@ where
         &self,
         validators: &BTreeMap<PublicKey, Bid>,
         auction_delay: u64,
-    ) -> BTreeMap<u64, SeigniorageRecipients> {
-        let initial_snapshot_range = INITIAL_ERA_ID..=INITIAL_ERA_ID + auction_delay;
+    ) -> BTreeMap<EraId, SeigniorageRecipients> {
+        let initial_snapshot_range = INITIAL_ERA_ID.iter_inclusive(auction_delay);
```

Comment:
We can't make
```rust
INITIAL_ERA_ID..=(INITIAL_ERA_ID + auction_delay)
```
work, can we? I think I looked that up some time and found that it's not yet possible in stable Rust, but now I can't find it anymore. :confused:

Reply:
Step trait is still nightly feature unfortunately https://doc.rust-lang.org/std/iter/trait.Step.html

Reply:
Ah, right, it was the `Step` trait! :+1: